matching-web/src/main/java/com/graphhopper/matching/http/MatchResultToJson.java

```diff
@@ -75,9 +75,9 @@ public JSONObject exportTo() {
 
             for (GPXExtension extension : edgeMatch.getGpxExtensions()) {
                 JSONObject wpt = new JSONObject();
-                wpt.put("x", extension.getQueryResult().getSnappedPoint().lon);
-                wpt.put("y", extension.getQueryResult().getSnappedPoint().lat);
-                wpt.put("timestamp", extension.getEntry().getTime());
+                wpt.put("x", extension.queryResult.getSnappedPoint().lon);
```

Comment:
Ok, I wasn't aware that GPXExtension is part of the public API. For me it was mainly the candidate representation but for the user it is the map matched GPX entry. I will then use getters for all fields in GPXExtension.